.virtualenv.dev-requirements.txt

```diff
@@ -50,3 +50,7 @@ python-dateutil
 
 # for release
 twine
+
+# for scripts testing
```

Comment:
This will cause testinfra & dataclasses to get pulled in into every development virtualenv. Unless you want that, I'd keep them separated as they are currently.